samples/bluetooth/observer/overlay_bbc_microbit-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,59 @@
+# Adjust Stack Sizes for reduce RAM usage
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_IDLE_STACK_SIZE=128
+CONFIG_ISR_STACK_SIZE=1024
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
+
+# Enable Extended Scanning
+# CONFIG_BT_EXT_ADV=y
+
+# Set maximum scan data length for Extended Scanning
+CONFIG_BT_EXT_SCAN_BUF_SIZE=192
+
+# Zephyr Bluetooth LE Controller needs 16 event buffers to generate Extended
+# Advertising Report for receiving the complete 1650 bytes of data.
+# Use 4 to be able to receive a minimal extended advertising with chains.
+CONFIG_BT_BUF_EVT_RX_COUNT=4
+
+# Use Zephyr Bluetooth Low Energy Controller implementation
+CONFIG_BT_LL_SW_SPLIT=y
+
+# Set maximum scan data length for Extended Scanning in Bluetooth LE Controller.
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=192
+
+# Increase Zephyr Bluetooth LE Controller Rx buffer to receive complete chain
+# of PDUs. Need 9 for maximum of 1650 bytes, but we use 3 as minimum to receive
+# at least a primary PDU, an auxiliary PDU and a chain PDU.
+CONFIG_BT_CTLR_RX_BUFFERS=3
+
+# Enable advanced features
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+
+# Adjust execution context priorities to achieve lower Radio ISR latencies
+CONFIG_BT_CTLR_LLL_PRIO=0
+CONFIG_BT_CTLR_ULL_HIGH_PRIO=1
+CONFIG_BT_CTLR_ULL_LOW_PRIO=1
+
+# Use just-in-time collision resolution in ticker and LLL pipeline
+CONFIG_BT_CTLR_LOW_LAT=n
+CONFIG_BT_CTLR_LOW_LAT_ULL_DONE=n
+CONFIG_BT_TICKER_LOW_LAT=n
+
+# Increase the below to receive interleaved advertising chains
+CONFIG_BT_CTLR_SCAN_AUX_SET=3
+CONFIG_BT_CTLR_LOW_LAT_ULL=y
+# CONFIG_BT_CTLR_SCAN_AUX_USE_CHAINS=y
+# CONFIG_BT_CTLR_SCAN_AUX_CHAIN_COUNT=3
+
+# Use unreserved timespace scanning
+CONFIG_BT_CTLR_SCAN_UNRESERVED=y
+
+# Code size reduction
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+CONFIG_LTO=y
+
+# Enable thread analysis
+CONFIG_THREAD_ANALYZER=y
+CONFIG_THREAD_ANALYZER_AUTO=y
+CONFIG_THREAD_ANALYZER_AUTO_INTERVAL=5
+CONFIG_THREAD_NAME=y
```

Comment:
Should this really be enabled by default? IIRC the bbc_microbit is fairly resource contrained, so it's odd to enable this for that board specifically.

Reply:
It was a chicken-or-egg problem when setting the stack size; with large values the firmware does not build or with low value it does not run. Hence, had to have the thread analysis enabled by default to ensure future (code or RAM usage changes require) tuning, is possible. If the firmware is running expected with thread analysis, then it can be disabled to save on code and RAM usage.

Reply:
I won't block the PR on this, and if @jhedberg and @HaavardRei both are fine with this, then it's OK :)

Reply:
Seems like they've now been moved to a new `debug.conf`